src/app/loans/loans-view/loan-account-actions/disburse-loan-account/disburse-loan-account.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="container">
+
+  <form [formGroup]="disbursementForm" (ngSubmit)="submit()">
+
+    <mat-card-content>
+
+      <div fxLayout="column">
+
+        <mat-form-field>
+          <mat-label>Disbursement On</mat-label>
+          <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="disbursementDatePicker" required formControlName="actualDisbursementDate">
+          <mat-datepicker-toggle matSuffix [for]="disbursementDatePicker"></mat-datepicker-toggle>
+          <mat-datepicker #disbursementDatePicker></mat-datepicker>
+          <mat-error *ngIf="disbursementForm.controls.actualDisbursementDate.hasError('required')">
+            Disbursement Date is <strong>required</strong>
+          </mat-error>
+        </mat-form-field>
+    
+        <mat-form-field>
+          <mat-label>Transaction amount</mat-label>
+          <input matInput required formControlName="transactionAmount">
+          <mat-error *ngIf="disbursementForm.controls.transactionAmount.hasError('required')">
+            Transaction Amount is <strong>required</strong>
+          </mat-error>
+        </mat-form-field>
+    
```

Comment:
Another field for 'Fixed EMI amount' is to be added in case the input data have a valid value for the key `fixedEmiAmount`.

Reply:
Is it in API doc? I'll check that

Reply:
Added Please review